libs/sdk-ui-filters/styles/scss/attributeFilterNext/attributeFilterDropdownBody.scss

```diff
@@ -10,3 +10,13 @@
         height: 100%;
     }
 }
+
+.gd-dashboards-root {
+    .gd-attribute-filter-dropdown-body__next {
+        // override the fixed width of the attribute filter dropdown body
+        // stylelint-disable-next-line declaration-no-important
+        width: auto !important;
```

Comment:
Do we have another way to do this?
As the best practice we should prevent use `important`!